### PR TITLE
Validating schedule

### PR DESCRIPTION
**Pull request by PierreSavatte**
This pull request address the issue #24

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -84,15 +84,37 @@ def test_it_creates_a_script(webapp):
...
     assert script.context.schedules[0].name == "every two minutes"
 
 
+def test_it_validate_schedule(webapp):
+    try:
+        schedule = Schedule(name="Schedule test", hour="/2", minute="/3")
+        db.session.add(schedule)
+        db.session.commit()
+    except ValueError:
+        assert True
```
Comment (ericgazoni) [inline tests/test_models.py]: you should check `pytest.raises`

File: tests/test_models.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def test_it_creates_a_job(webapp):
     context = Context(environment=test_env)
     context.set_config(url="http://localhost:8000", loop=30, retry=True)
 
-    schedule = Schedule(name="every two minutes", hour="*", minute="/2")
+    schedule = Schedule(name="every two minutes", hour="*", minute="*/2")
```
Comment (ericgazoni) [inline tests/test_models.py]: can you also fix the demo script please?

File: kirby/models/__init__.py [hunk 3]
```diff
@@ -150,6 +150,26 @@ class Schedule(db.Model):
     def add_suspension(self, suspension):
         self.suspensions.append(suspension)
 
+    @validates("hour", "minute", "day", "month", "weekday")
```
Comment (ericgazoni) [inline kirby/models/__init__.py]: According to my experience, `validates` only operates during INSERT and not during UPDATE, so I could input something correct, but break it afterwards. Have you tested updating?

File: tests/test_models.py (hunk 1 above)
Comment (PierreSavatte) [reply tests/test_models.py]: Now using `pytest.raises`

File: kirby/models/__init__.py (hunk 3 above)
Comment (PierreSavatte) [reply kirby/models/__init__.py]: I added a test on UPDATE :

```python
    [...]
    schedule = (
        db.session.query(Schedule).filter_by(name="test schedule").one()
    )
    with raises(ValueError):
        schedule.minute = "/3"
        db.session.commit()
```

The test pass and raise an error when the `raises` is absent, so it is working.

File: tests/test_models.py (hunk 2 above)
Comment (PierreSavatte) [reply tests/test_models.py]: Resolved

**Pull request merged**